src/API/Rest/InferenceRequest.cs

```diff
@@ -257,32 +260,35 @@ public bool IsValid(out string details)
                 errors.Add("No algorithm defined or more than one algorithms defined in 'inputResources'.  'inputResources' must include one algorithm/pipeline for the inference request.");
             }
 
-            if(InputMetadata?.Details is null)
+            if (InputMetadata?.Details is null)
             {
                 errors.Add("Request has no `inputMetadata` defined.");
             }
             else
             {
-                switch(InputMetadata.Details.Type)
+                switch (InputMetadata.Details.Type)
                 {
                     case InferenceRequestType.DicomUid:
                         if (InputMetadata.Details.Studies.IsNullOrEmpty())
                         {
                             errors.Add("Request type is set to `DICOM_UID` but no `studies` defined.");
                         }
                         break;
+
                     case InferenceRequestType.DicomPatientId:
-                        if(string.IsNullOrWhiteSpace(InputMetadata.Details.PatientId))
+                        if (string.IsNullOrWhiteSpace(InputMetadata.Details.PatientId))
                         {
                             errors.Add("Request type is set to `DICOM_PATIENT_ID` but `PatientID` is not defined.");
                         }
                         break;
+
                     case InferenceRequestType.AccessionNumber:
-                        if(InputMetadata.Details.AccessionNumber.IsNullOrEmpty())
+                        if (InputMetadata.Details.AccessionNumber.IsNullOrEmpty())
```

Comment:
wait -- how can this work? if `InputMetadata.Details.AccessionNumber` is null, you cannot `.IsNullOrEmpty()` 😕

Reply:
It's an extension method.